.eslintrc.yml

```diff
@@ -645,3 +651,24 @@ rules:
     - error
   lodash/prefer-times:
     - error
+
+  # JSDoc
+
+  jsdoc/check-param-names:
+    - error
+  jsdoc/check-tag-names:
+    - error
+  jsdoc/newline-after-description:
+    - error
+  jsdoc/require-example:
```

Comment:
Do we really want to force an example for *every* function & method? Seems to me that'd be a bit of overkill, and introduce a lot of noise in the code.